### DISCUSSION TITLE
How can I a read in a DEF from a previous run and then route it?

### DISCUSSION THREAD

**Discussion by jarvis-lorry** [The-OpenROAD-Project/OpenLane#1256] [Q&A] [answered]
I have a DEF saved from a design run that I did with an earlier version of OpenLane and I would like to read it in to OpenLane right now and have it rerouted and completed.  Is there an easy way to simply read in a DEF, have OpenLane keep CTS elements and cell placement but ignore all of the metal lines, replacing them with new routed lines?

**Comment by vijayank88** [comment, marked as answer]
@jarvis-lorry 
If you're using through interactive mode use:
`set ::env(CURRENT_DEF) path2DEF.def`

**Comment by jarvis-lorry** [comment]
Thank you, @vijayank88.  I've been able to bring in a different DEF file after running the interactive steps up to CTS.
